### ISSUE TITLE
No features show except text

### ISSUE THREAD

**Issue by Wizorx** [sthewissen/Xamarin.Forms.PancakeView#136]
I'm new to xamarin so sorry in advance for any silly mistakes. I followed [the tutorial](https://dev.to/petermilovcik/xamarin-forms-how-to-use-pancakeview-1lch) but still only the text appears on my preview, no error codes are thrown. I'm in the latest versions of Xamarin Forms, VS 2019 and Pancake View.
![image](https://user-images.githubusercontent.com/20863671/93814132-80ec0980-fc2a-11ea-8c9e-b436b7d2c166.png)

**Comment by sthewissen** [comment]
`BackgroundGradientAngle`, `BackgroundGradientStartColor` and `BackgroundGradientEndColor` are all deprecated in the latest released version. Check out the documentation on doing background gradients to see the new way of doing it.

https://github.com/sthewissen/Xamarin.Forms.PancakeView/wiki/Gradients

**Comment by Wizorx** [comment]
Still nothing
![image](https://user-images.githubusercontent.com/20863671/93926736-464ba500-fcee-11ea-80cb-d9c3f0aa3099.png)

**Comment by sthewissen** [comment]
What happens when you put something like a `Grid` around it? I must admit I've never tried making it the root element of the page. Also, is this AndroidX?

**Comment by Wizorx** [comment]
Nothing happens, still plain text with some padding.
![image](https://user-images.githubusercontent.com/20863671/94033128-0987b880-fd97-11ea-8721-87f17afee6b3.png)

**Comment by sthewissen** [comment]
Is this project shareable somewhere so I could take a look?

**Comment by sthewissen** [comment]
Also, just now, I noticed that in that Android projects references there's no PancakeView package. It needs to be installed into both the Shared project and the platform-specific projects you want to support.

**Comment by Wizorx** [comment]
Makes sense, I'll try adding to the Android project, if it doesn't work out I'm gonna upload it somewhere and share the link here so you can take a look.

**Comment by Roci-dev** [comment]
I'm having this same issue. I have it installed to the project level and android and the only thing that displays is a background color (no gradients) and text.
